test/components/util/util.go

```diff
@@ -199,10 +213,60 @@ func WaitForDeployment(t *testing.T, client kubernetes.Interface, ns, name strin
 		t.Logf("all pods for deployment %s, are in ready state", deploy.Name)
 		return true, nil
 	}); err != nil {
+		if err := printPodLogs(t, client.CoreV1().Pods(ns), deploy.Spec.Selector); err != nil {
+			t.Logf("reading pod logs: %v", err)
+		}
+
 		t.Errorf("error while waiting for the pods: %v", err)
 	}
 }
 
+// printPodLogs print logs from all pods and containers selected by given selector.
+func printPodLogs(t *testing.T, p corev1typed.PodInterface, selector *metav1.LabelSelector) error {
+	labelMap, err := metav1.LabelSelectorAsMap(selector)
+	if err != nil {
+		return fmt.Errorf("converting label selector to map: %w", err)
+	}
+
+	pods, err := p.List(context.TODO(), metav1.ListOptions{LabelSelector: labels.SelectorFromSet(labelMap).String()})
```

Comment:
same can be done here https://github.com/kinvolk/lokomotive/pull/765/files#r466315360

Reply:
Done, thanks.